File: baekjoon/java/src/bronze5/Q10869_FourOperations.java
package bronze5;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Scanner;
import java.util.StringTokenizer;

public class Q10869_FourOperations {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String line = br.readLine();
        StringTokenizer stk = new StringTokenizer(line, " ");
        int a = Integer.parseInt(stk.nextToken());
        int b = Integer.parseInt(stk.nextToken());

        System.out.println(a + b);
        System.out.println(a - b);
        System.out.println(a * b);
        System.out.println(a / b);
        System.out.println(a % b);
        // 14268KB 104ms
    }

    private static void useBufferedReader() throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String line = br.readLine();
        String[] splitInput = line.split(" ");
        int a = Integer.parseInt(splitInput[0]);
        int b = Integer.parseInt(splitInput[1]);

        System.out.println(a + b);
        System.out.println(a - b);
        System.out.println(a * b);
        System.out.println(a / b);
        System.out.println(a % b);
        // 14344KB 104ms
    }

    private static void inputByScanner() {
        Scanner scanner = new Scanner(System.in);
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        scanner.close();

        System.out.println(num1 + num2);
        System.out.println(num1 - num2);
        System.out.println(num1 * num2);
        System.out.println(num1 / num2);
        System.out.println(num1 % num2);
        // 17720KB 168MB
    }
}

File: baekjoon/kotlin/src/main/kotlin/bronze5/Q10869_FourOperations.kt
package botech.bronze5

import java.io.BufferedReader
import java.io.InputStreamReader

class Q10869_FourOperations

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val input = br.readLine();
    val inputArray = input.split(" ")
    val a = inputArray[0].toInt()
    val b = inputArray[1].toInt()
//    val (a, b) = input.split(" ").map { it.toInt() } // 21276KB, 108ms

    println(a + b)
    println(a - b)
    println(a * b)
    println(a / b)
    println(a % b)

    br.close()
    // 14300KB 88ms
}
